sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -369,8 +366,8 @@ def _store_side_inputs_in_state(self,
           state_key = beam_fn_api_pb2.StateKey(
               iterable_side_input=beam_fn_api_pb2.StateKey.IterableSideInput(
                   transform_id=transform_id, side_input_id=tag, window=window))
-          bundle_context_manager.worker_handler.state.append_raw(
-              state_key, elements_data)
+          runner_execution_context.worker_handler_manager.state_servicer\
```

Comment:
Perhaps add a state_servicer() method right on runner_execution_context (and use several places below)?

Reply:
On second thought, perhaps the ownership of state servicer should be moved up to runner_execution_context (though the worker manager may need a reference). Your call.

Reply:
I've just added the state_sevicer() from your first comment, but I've created a jira issue to track fixng that later on.

Reply:
Link to JIRA for refernce?